pkgs/servers/nextcloud/packages/31.json

```
@@ -0,0 +1,422 @@
+    "homepage": "",
+    "licenses": [
+      "agpl"
+    ]
+  },
+  "mail": {
+    "hash": "sha256-8/wicUZHtPRwBlLU5PYXvSR8mPo7q9iaeaMFmbvEdMA=",
+    "url": "https://github.com/nextcloud-releases/mail/releases/download/v4.2.1/mail-stable4.2.tar.gz",
+    "version": "4.2.1",
+    "description": "**💌 A mail app for Nextcloud**\n\n- **🚀 Integration with other Nextcloud apps!** Currently Contacts, Calendar & Files – more to come.\n- **📥 Multiple mail accounts!** Personal and company account? No problem, and a nice unified inbox. Connect any IMAP account.\n- **🔒 Send & receive encrypted mails!** Using the great [Mailvelope](https://mailvelope.com) browser extension.\n- **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://horde.org) libraries.\n- **📬 Want to host your own mail server?** We do not have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!\n\n## Ethical AI Rating\n\n### Priority Inbox\n\nPositive:\n* The software for training and inferencing of this model is open source.\n* The model is created and trained on-premises based on the user's own data.\n* The training data is accessible to the user, making it possible to check or correct for bias or optimise the performance and CO2 usage.\n\n### Thread Summaries (opt-in)\n\n**Rating:** 🟢/🟡/🟠/🔴\n\nThe rating depends on the installed text processing backend. See [the rating overview](https://docs.nextcloud.com/server/latest/admin_manual/ai/index.html) for details.\n\nLearn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).",
+    "homepage": "https://github.com/nextcloud/mail#readme",
+    "licenses": [
+      "agpl"
+    ]
+  },
+  "maps": {
+    "hash": "sha256-/uoM29jXqeOvOJBu3xhv+KgqPE7T03pV269fdr2Er+0=",
+    "url": "https://github.com/nextcloud/maps/releases/download/v1.6.0-2-nightly/maps-1.6.0-2-nightly.tar.gz",
+    "version": "1.6.0",
```

Hah, I almost forgot!

I had to disable this since this doesn't support Nextcloud 31. Isn't nc4nix supposed to detect this?

---

@transcaffeine have you by chance created 31.json from 30.json before running `generate.sh`? nc4nix only checks for updates after that and since there are no releases compatible with v31, it seems as if it's left untouched :melting_face:  

`maps` is incompatible with 31 (this makes `nextcloud-setup.service` fail hard), so I recreated the file and now it's not part of it anymore, as expected.

---

Interesting - no, i ran `generate.sh` and it (`nc4nix1`) created the `31.json` newly.

---

Hmm, weird. Perhaps they changed the metadata in between or what not.
I should take a closer look at this again at some point, this isn't the first time, weird things happen with it.